### ISSUE TITLE
PeoplePicker: [a11y] [ScreenReader] when navigation suggestion list, better to read something well structured.

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#1200]
### Bug Report

- __Package version(s)__:  latest
- __Browser and OS versions__: IE + JAWS

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No, don't know an easy way to implement.

Requested priority: Low

#### Describe the issue:

1. When using people picker, wait for the suggestion box popping up.
2. Use up and down arrow keys to navigate.
3. With JAWS screen reader, it reads <name> <job> <email> which is the information of one user.
4. Better to have structured string:
` **The name is** <name>. **The job is** <job>. **The email is** <email>. `
So blind people will have better experience and understanding of the suggestion item.

#### Actual behavior:

JAWS read `<name> <job> <email>`

#### Expected behavior:

JAWS read `**The name is** <name>. **The job is** <job>. **The email is** <email>.`

**Comment by ipip2005** [comment]
An applicable fix would be expose an callback which takes person info as argument and returns the screen reader string for each item. If consumer doesn't override it, then use the default combination: name/job/email

**Comment by hross** [comment]
I tried [overriding the aria-label of the PeoplePicker](http://codepen.io/pogostick_snowman/pen/evWLVM?editors=0011) but unfortunately the screen reader doesn't read it (NVDA) so I think to support this you would need to override the whole element.

**Comment by hross** [comment]
@ipip2005 please take a look at the merged PR. Does this solve your issue by allowing you to override the text?

**Comment by ipip2005** [comment]
@hross 
I can override the text, but I want to read different text to screen readers.
For example below persona card
![image](https://user-images.githubusercontent.com/7711735/27782528-79eb5b8c-6005-11e7-9304-64c7f6863c0a.png)

It displays
```
Liaoliang Ye
SOFTWARE ENGINEER
```
But I want screen reader to read `Name: Liaoliang Ye, Job: SOFTWARE ENGINEER`  when I select any of the suggestions in PeoplePicker.

I suppose the fix could be exposing aria-label in SuggestionItem, then PeoplePicker should be able to take aria-labels from consumers and pass them to Suggestionitems.
IMO, the key is assigning an appropriate aria-label to the actual `focusable` item which is the suggestion item I suppose
